CoreGraphNode/nbproject/project.xml

```diff
@@ -6,6 +6,14 @@
             <code-name-base>au.gov.asd.tac.constellation.graph.node</code-name-base>
             <suite-component/>
             <module-dependencies>
+                <dependency>
```

Comment:
FYI an aspiration idea behind some of our modules like `CoreGraphNode` or `CorePluginFramework` is that it is portable. Adding more dependencies needs to be considered carefully. In this case I don't have an issue as it is 3rd party jars. `CoreGraphNode` is also heavily linked to NetBeans now anyway...

Reply:
I can't think of why it would have made it's way into the PR - Maybe when I added an import and just updated a dependency.